cub/test/catch2_test_device_scan.cu

```diff
@@ -33,12 +33,29 @@
 #include <cstdint>
 
 #include "c2h/custom_type.cuh"
-#include "c2h/extended_types.cuh"
 #include "catch2_test_device_reduce.cuh"
 #include "catch2_test_device_scan.cuh"
 #include "catch2_test_helper.h"
 #include "catch2_test_launch_helper.h"
 
+// Circumvate DECLARE_LAUNCH_WRAPPER appending stream as the last argument
+// by default since init_value needs to be the last argument.
+template <typename InputIteratorT, typename OutputIteratorT, typename ScanOpT, typename InitT>
+CUB_RUNTIME_FUNCTION static cudaError_t inclusive_scan_init_wrapper(
+  void* d_temp_storage,
+  size_t& temp_storage_bytes,
+  InputIteratorT d_in,
+  OutputIteratorT d_out,
+  ScanOpT scan_op,
+  InitT init,
+  int num_items,
+  cudaStream_t stream = 0)
+{
+  return cub::DeviceScan::InclusiveScanInit(
+    d_temp_storage, temp_storage_bytes, d_in, d_out, scan_op, init, num_items, stream);
+}
```

Comment:
Question: I am confused, because `InclusiveScanInit` takes `stream` as the last argument, as it should. Why is this necessary?

Reply:
I completely deleted this wrapper, you were right, thanks a lot.